packages/terra-section-header/src/SectionHeader.jsx

```diff
@@ -13,7 +13,7 @@ const propTypes = {
   /**
    * Text to be displayed on the SectionHeader.
    */
```

Comment:
```suggestion
  /**
   * Text to be displayed on the SectionHeader. 
   * *Note: `text` prop is required for section-header.
   */
```

Reply:
Text prop made required in https://github.com/cerner/terra-core/pull/3672/commits/b0ad24f2ab2f6797cec1a520ce4b1443638375b9